src/components/Calendar.js

```diff
@@ -2,11 +2,14 @@ const React = require('react');
 const PropTypes = require('prop-types');
 const Radium = require('radium');
 const moment = require('moment');
+const keycode = require('keycode');
 
 const Icon = require('./Icon');
 
 const { themeShape } = require('../constants/App');
 
+import { __calculateDayByKey } from '../enhancers/calendar-enhancers';
```

Comment:
since we call these `utils` in our main codebase, maybe stick to that naming convention in the open source?

Reply:
sounds good - i thought of this idea of trying to keep our actual components very minimal (rendering JSX only, if possible) and moving a lot of these functions into utils. 

i guess what I'm saying is I think we under-use utils for this sort of thing. writing code like this will help keep our functions more pure and easier to test